packages/playground-examples/copy/id/JavaScript/JavaScript Essentials/Code Flow.ts

```diff
@@ -0,0 +1,65 @@
+//// { order: 3, compiler: { strictNullChecks: true } }
+
+//// {order: 3, compiler: {strictNullChecks: true}}
+
+// Bagaimana kode mengalir di dalam file JavaScript dapat mempengaruhi
+// tipe (type) di seluruh program kita.
+
+const users = [{ name: "Ahmed" }, { name: "Gemma" }, { name: "Jon" }];
+
+// Kita akan melihat apakah kita dapat menemukan pengguna bernama "jon".
+const jon = users.find(u => u.name === "jon");
+
+// Dalam kasus di atas, 'find' bisa gagal. Dalam hal ini kita
+// tidak memiliki objek. Ini menghasilkan sebuah tipe (type):
+//
+// {name: string} | undefined
+//
+// Jika Anda mengarahkan mouse ke tiga penggunaan 'jon' berikut,
+// Anda akan melihat bagaimana tipe berubah tergantung di mana kata tersebut berada:
```

Comment:
tipe di sini kalau dalam konteks bahasa inggrisnya kan `type`, mungkin bisa dipadankan menjadi `tipe data`?

Reply:
@jefrydco alright, on the way recheck & fix. 👍 🚀